README.md

```diff
@@ -1935,3 +1942,4 @@ Hello, it`s [me](https://github.com/Labsik)
 
 ---
 
+
```

Comment:
But this new line is unnecessary.

Reply:
Thanks for the review! I see that line 1945 is empty now.